arborist/auth.go

```diff
@@ -91,6 +93,7 @@ type AuthRequest struct {
 	ClientID string
 	Policies []string
 	Resource string
+	Tag      string
```

Comment:
I think you can remove this line if we're not supporting `tag` as optional field